Gemfile

```diff
@@ -52,11 +52,13 @@ end
 group :development, :test do
   gem "m", "~> 1.5", require: false
   gem "pry-byebug"
-  gem "rubocop", require: false
-  gem "rubocop-rails", require: false
-  gem "rubocop-performance", require: false
-  gem "brakeman", require: false
   gem "toxiproxy", "~> 1.0.0"
+
+  # please update .github/workflows/lint.yml on versions update
```

Comment:
could it be possible to move following to a group and install only these with `--without`? I am assuming this change is being made because we don't want to run entire `bundle install` on lint jobs.

Reply:
Yes, I tried to avoid full bundle. But any idea how to do that? Gems are by default in "default" group if I remember well. Should I name all groups except that one?

Reply:
Anyway I think we can move to full bundle, since it will get cached to avoid this problem.

Reply:
I think bundle --without default,test,development,assets should work. but yeah, if it is cached, probably using complete bundle install should also not be an issue. 
I was just hoping we don't have to maintain the dependencies list at two places since dependabot updates these.

Reply:
I have moved to full bundle for now. I'll check how complex would be to implement https://github.com/rubygems/rubygems/issues/4048 and potentially we can later opt-in only specified groups to get improved build time for linters.

Reply:
This means we can remove the comment right?

Reply:
we have disabled dependabot update on rubocop*, so version pinning is fine for them. However please remove it from brakeman, or else it won't be automatically updated.

Reply:
done